students_exercises/mitesh_student_dao.py

```diff
@@ -0,0 +1,77 @@
+# Insert Staements(Insert random 100 students and then insert specific students)
+
+def utility(studentinfo):
+	st = ""
+	for key in studentinfo.keys():
+		if key == 'name':
+			st = st + key + " = '" + studentinfo[key] + "', "
+		elif key == 'id':
+			continue
+		else:
+			st = st + key + " = " + str(studentinfo[key]) + ", "
+	print "---------------- ", st
+	return st
+	
+def insertrandomstudents(number):
+	#st = ""
+	for count in range(1, number):
+		st = "INSERT INTO student (id, name, age) VALUES ('" + str(count) + "', " + "'name_" + str(count) + "', '" + str(count) + "');"
+		print st
+		
+		
+insertrandomstudents(10)
+	
+studentinfo = {'name': 'mitesh', 'age': 30, 'id': 1, 'phnumber':123}
+
+def insertspecificstudent(studentinfo):
+	st = "INSERT INTO student ("
+	for key in studentinfo.keys():
+		st = st + key + ", "
+	
+	st1 = st[:-2] + ") VALUES ("
+	for key in studentinfo.keys():
```

Comment:
can we make this as generic utility?

Reply:
I dont understand what you are saying?
Can u please elaborate more?